packages/core/src/metrics/utils.ts

```diff
@@ -54,15 +53,52 @@ export function serializeMetricBuckets(metricBucketItems: MetricBucketItem[]): s
   return out;
 }
 
+/** Sanitizes units */
+export function sanitizeUnit(unit: string): string {
```

Comment:
Can we add docstring + link back to develop docs for these `sanitizeUnit` function?